digitalpy/core/telemetry/impl/opentel_metrics_provider.py

```diff
@@ -1,6 +1,6 @@
 from digitalpy.telemetry.impl.opentel_meter import OpenTelMeter
 from digitalpy.telemetry.impl.opentel_metric_reader import OpenTelMetricReader
-from digitalpy.telemetry.metrics_provider import MetricsProvider
+from digitalpy.core.telemetry.metrics_provider import MetricsProvider
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I001 isort found an import in the wrong position